specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-04-01-preview/managedservices.json

```diff
@@ -0,0 +1,855 @@
+        },
+        "display": {
+          "readOnly": true,
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "type": "string",
+              "description": "Service provider: Microsoft.ManagedServices"
+            },
+            "resource": {
+              "type": "string",
+              "description": "Resource on which the operation is performed: Registration definition, registration assignment etc."
+            },
+            "operation": {
+              "type": "string",
+              "description": "Operation type: Read, write, delete, etc."
+            },
+            "description": {
+              "type": "string",
+              "description": "Description of the operation."
+            }
+          }
+        }
+      },
+      "type": "object",
+      "description": "Object that describes a single Microsoft.ManagedServices operation."
+    },
+    "OperationList": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of Microsoft.ManagedServices operations."
+        }
+      },
+      "type": "object",
+      "description": "List of the operations."
+    },
+    "Authorization": {
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "Principal Id of the security group/service principal/user that would be assigned permissions to the projected subscription"
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The role definition identifier. This role will define all the permissions that the security group/service principal/user must have on the projected subscription. This role cannot be an owner role."
+        }
+      },
+      "required": [
+        "principalId",
+        "roleDefinitionId"
+      ],
+      "type": "object",
+      "description": "Authorization tuple containing principal Id (of user/service principal/security group) and role definition id."
+    },
+    "ErrorResponse": {
+      "properties": {
+        "error": {
+          "readOnly": true,
+          "properties": {
+            "code": {
+              "type": "string",
+              "description": "Error code."
+            },
+            "message": {
+              "type": "string",
+              "description": "Error message indicating why the operation failed."
+            }
+          },
+          "required": [
+            "code",
+            "message"
+          ],
+          "type": "object",
+          "description": "Error response indicates Azure Resource Manager is not able to process the incoming request. The reason is provided in the error message."
+        }
+      },
+      "description": "Error response."
+    },
+    "OperationStatus": {
```

Comment:
>OperationStatus [](start = 5, length = 15)

where is this used ?

Reply:
you missed to remove this after removing operationstatus api

---
In reply to: [289587436](https://github.com/Azure/azure-rest-api-specs/pull/5947#discussion_r289587436) [](ancestors = 289587436)

Reply:
Oh yes, I forgot to remove. Removed now :)